### ISSUE TITLE
Default dashboard keeps resetting to Overview

### ISSUE THREAD

**Issue by GitHubGoody** [home-assistant/frontend#18626]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I recently reconfigured my primary custom .yaml dashboard configuration to clean up the URL (i.e. remove reference to anything "lovelace"). My configuration now looks like this:

```
lovelace:
  mode: storage
  dashboards:
    dash-board:
      mode: yaml
      filename: dashboard.yaml
      title: Home
      icon: mdi:home-account
      show_in_sidebar: true
      require_admin: false
```
Ever since, some of my devices seem to reset the default dashboard every 1-2 days to the Overview which is marked as default even if I stay logged in and on WiFi and/or when the app has to switch to external access (Nabu Casa) and then back to internal.

I cannot seem to pinpoint what part of the configuration changes are causing this. Any ideas or is this just a bug I need to report?

### Describe the behavior you expected

The dashboard I set as the default to persist.

### Steps to reproduce the issue

See above

### What version of Home Assistant Core has the issue?

2023.11.2

### What was the last working version of Home Assistant Core?

N/A

### In which browser are you experiencing the issue with?

Multiple including iOS HA app and Edge

### Which operating system are you using to run this browser?

iOS 17.0.3 and Windows 10

**Comment by GitHubGoody** [comment]
More context added [here](https://community.home-assistant.io/t/default-dashboard-keeps-resetting-to-overview/656482).

**Comment by janrivo** [comment]
I also experience this. From what I can tell the defaultPanel in the local storage is reset on a refresh of the page, but I don't know why this happens.

**Comment by jprokopowich** [comment]
This occurs on my android device, ios device and desktop.

**Comment by prohidium** [comment]
Can you clarify if this is just in your user setting for your own default dashboard or is it actually changing the default dashboard outside of your own user default dashboard under user settings?

I have the problem which appears to be related to the above complaint which is triggered by connecting to a different network for example: I am away from home and on LTE or a different network and I connect via the iphone APP using NabuCasa. My user profile defaults to the HA default dashboard instead of my user default dashboard. I then have to go into my user profile and change it back to my actual default dashboard which is a different dashboard than the Overview default dashboard. My default in HA is not the Overview default dashboard but rather a dashboard I created.

**Comment by GitHubGoody** [comment]
Well, with all the changes I made trying to put things back to default setting, my dashboards are still not displaying as intended and my problem now is that I cannot seem to get the Overview page to stay in the sidebar (ironically, the inverse symptom of my original issue).

I've returned my configuration.yaml back to the default setting for a [YAML Overview dashboard](https://www.home-assistant.io/dashboards/dashboards/#using-yaml-for-the-overview-dashboard):

```
lovelace:
  mode: yaml #I have also tried storage
  resources: !include resources.yaml
```
But, unless I manually select Overview on each device, it doesn't default to Overview. Also, even when I manually set it per device (via Profile>Dashboard **OR** Settings>Dashboard>Overview>SET AS DEFAULT ON THIS DEVICE), it seems any number of things (*including force closing and reopening the iOS app while still on my home network OR turning off WiFi on my iPhone forcing a refresh) can cause it to still disappear from the sidebar. In both cases, if I go back into Settings>Dashboard, _Show in sidebar_ is once again deselected and none of the dashboards are selected as default (same under Profile). **So, in this way, this is still symptomatic of my original issue which is that the default device dashboard keeps resetting (previously to Overview and now to null).**

So, instead, I've had to create a duplicate dashboard (title Dashboard) that is visible in the sidebar and does seem to hold as a default. Here is that part of my configuration...also under lovelace:
```
  dashboards:
    dash-board:
      mode: yaml
      filename: dashboard.yaml
      title: Dashboard
      icon: mdi:home-account
      show_in_sidebar: true
      require_admin: true
```

Fortunately, my configuration is split, so I just had to duplicate the reference ui-lovelace.yaml file to dashboard.yaml which only contains

`views: !include_dir_merge_list views`

Here is a look at my current Dashboard summary _from my laptop_:

![image](https://github.com/home-assistant/frontend/assets/46235745/bbc6ecd6-6d22-4d41-ad0f-e4d41939364d)

*If this were on my iPhone, the two checkmarks on the Overview line would disappear if I did either of the things mentioned.

I'm open to any ideas on how to get back to a stable dashboard configuration since this affects all of the members of my family. It's relatively easy (but still a PITA) for me to get a dashboard back up every.single.time since I'm an Admin, but the rest of my family couldn't get to any of the HA features until I implement the duplicate Dashboard workaround.

**Comment by akropp** [comment]
This happens to me periodically as well.  It will save for a while, but eventually will revert back to Overview.

**Comment by codemunkie15** [comment]
This is still an issue at least on iOS for me.

**Comment by akropp** [comment]
Still an issue for me as well.

**Comment by prohidium** [comment]
Happened to me just a day or two ago on my iphone. Each time this happens, I have to go into my user profile within the iphone app and reset my dashboard back to the correct one that I have set as default within HA. It appears that the HA companion app still sees Overview as the default dashboard periodically even though I have set a different dashboard as the default dashboard within HA.

**Comment by ildar170975** [comment]
Can it be explained by loosing a local settings?
Create a simple custom theme, apply it on a Companion app - and see if this setting is persistent. If the them is reset to "default" - here is a possible reason.

**Comment by akropp** [comment]
Possibly, but sort of beside the point.  There really needs to be a way to
set the default dashboard on the server side.  The "Overview" dashboard is
totally worthless in a big installation, and actually crashes the ios app
sometimes.  So whether it is the app resetting itself to the default, or
giving a new user access, there should be a way to set which dashboard the
app starts on.  All of us maintaining these installations are probably fine
with tinkering and adjusting settings, but if we want buy-in from less
technical people (e.g. my wife), it needs to be pretty seamless.  I have
not installed the app for her partly because of the inevitable "what the
hell is this?" when the app comes up frozen on the overview page.

--Adam

**Comment by prohidium** [comment]
Ok, I think I may have found the issue. In the companion app, along with setting the default dashboard within your profile, in settings/dashboards, you also have to select your desired dashboard and “set it to default for this device”.  I have never set the default on each device and just assumed that my default dashboard set within my profile in the companion app would take care of it. When I switch from wifi (internal access) to LTE (external NabuCasa access), it must dump the profile settings. 

So, on the companion app on each iOS device, go to settings/dashboard and make the dashboard you want as default and then also go into your login profile in the companion app and set your default dashboard there as well and I think all will be good going forward.

**Comment by loopy321** [comment]
I have the same issue of resetting to "Overview" both in iOS app and browser.  Right now, I open a Chrome incognito window (no cookies), login as user, click on user icon, set default dashboard to other than "Overview", reload page and verify it works as expected.  I then log the user out and log back in - _bam_, back to overview, and the dashboard setting I just changed is back to "Overview".  
Something is obviously broken here...

**Comment by karwosts** [comment]
This is simply by design currently, and the only way it could be different is with a new feature request. Please check and see if it exists in the feature requests forum (I'm sure it does) and vote for it there. 

There is no ability to set a default dashboard per user. There is only the ability to set a default dashboard for a specific client, and all client settings are reset on logout or cache cleared. So that's expected behavior unfortunately.

**Comment by jhemak** [comment]
Not stale

**Comment by dafi87** [comment]
This is still an ongoing issue. I need a different default dashboard for mobile devices, but whatever I set as default dashboard in the iOS app reverts back to Overview after a day or just some minutes.
It's honestly so so frustrating because you run into this problem multiple times a day and it definitely decreases the overall acceptance factor significantly!

Please give us a solution for this or tell us how to help developing one.
Thank you!

**Comment by xpac1985** [comment]
I've now created a matching feature request for this - if you feel like voting today ;-)

https://community.home-assistant.io/t/make-default-dashboard-and-other-settings-persistent-per-device/828865

**Comment by radupantiru** [comment]
I am experiencing the same issue on Linux Firefox, both Android and iOS apps. Quite frustrating.

**Comment by mbino** [comment]
Still have this issue.

**Comment by akropp** [comment]
Definitely still an issue.  I don't understand why given the whole point of Home Assistant is that you can customize everything however you want, you are always forced to show your users this default dashboard and have to instruct them how to get to the actual dashboard you want them to use.  Particularly since it is completely broken for large installations.  It sets a very bad first impression to new users.

**Comment by freddeh** [comment]
Still an issue and definitely has nothing to do with logging out. I have this issue frequently (e.g. after updating Home Assistant via docker) or when being outside of my home WiFi.

**Comment by bbostock** [comment]
That does not have any positive effect. I have now given in gracefully and copied my preferred dashboard into Overview. It would be nice if i could change the name and icon.

**Comment by ppiotrek90** [comment]
I also experience the same issue. It has started only recently - can’t say when exactly but I guess couple weeks ago. It happens on two iPhones that are connected to HA. All devices as well as HA are up to date. 

Is there any workaround? 

iOS 18.6.1
iPhone 15 Pro + iPhone 15

HA Core 2025.8.3
HA 16.1

**Comment by tparchem** [comment]
Chiming in to say I also am experiencing this issue.

iOS 18.6.2
Companion App 2025.7.2 (2025.1356)
HA Core 2025.08.3
HA Supervisor 2025.8.3
HA OS 16.1
Frontend 20250811.1

**Comment by agnarvold** [comment]
Same problem on two iPhones. iOS, HA and companion app are all updated with latest updates.

**Comment by SebastienLemayeur** [comment]
Same problem here, also on two iPhones. Everything up to date. Horrible for girlfriend acceptance factor unfortunately.

**Comment by tonydm** [comment]
Save or similar issue

Logging in any user, Admin or not, Selecting Default Dashboard and Theme for the user on a specific device does not persist upon logging out and back in... All user settings revert... default dashboard is back to Overview, theme is back to default.

I have tried, as the suggested method, to temporarily change the user to an Administrator, logging in as the user, setting default dashboard and theme.  Log out the user.  Log in as myself, set the user back to a non Administrator.  Log back in as user and all settings have reverted back to default dashboard of Overview, theme default, etc.  All these taking place on the same device in the same browser (no extensions or other third party interference).

I have tried this on Chrome, Firefox, Brave (again, no extensions to interfere)... all result in the same behavior.  Very frustrating to say the least!.  I ask, what's the point in being able to set the Default Dashboard and Theme when these settings don't remain saved?  It's not an uncommon activity to log out and back in to a website, we do it every single day in our online website activities.  I appreciate the per device state, nothing wrong with that.  But the defaults which are set on a given "device" must remain as they are set/configured.

For context, I have set up a "Home" Dashboard with Admin Only or user visibility restrictions.  It is set as the Default Dashboard under the Dashboards globally.  There are no cards on the Home Dashboard that have visibility restrictions.  I try and set up a User with the Home Dashboard as the default dashboard,  As described, the settings revert upon logout and login.  Not how I would expect it to work.

If I have missed something in my review of the numerous posts found on the Home Assistant forum and this Github Issue with this problem of the Default Dashboard being set but not persisting on a log out/log in, or if have a misunderstanding of how the user settings must be set up, I would be appreciative for the clarification and a solution

Thank you

**Comment by tonydm** [comment]
I apologize, I guess I didn't understand the purpose of pointing to another Issue thread (https://github.com/home-assistant/iOS/issues/3783) that simply points back to this Issue thread.  I was simply confused about your reply to my post.  I interpreted your post as a link to a possible solution which only pointed back to this Issue.  So I was confused.  But either way, it's not a big deal. :)

**Comment by LarsStegman** [comment]
GitHub automatically does that when you mention one issue in another. I just pointed from the iOS issue to this one and GitHub created the pointer the reverse way.

**Comment by imc67** [comment]
Same here on several iOS and iPadOS devices, very annoying because there are “non-tech-users” at home who don’t understand the “default” dashboard with an overview of hundreds if not thousands of all sensors, switches etc. 😔

**Comment by Coliinnn** [comment]
Same issue here and on my partners phone but unable to specifically replicate this. I understand it's not very helpful to keep posting "same", but attention for this issue would be appreciated since it is detrimental for usability and partner approval. Very curious if anyone has an idea of where to start looking for a fix?

**Comment by gNeeUs** [comment]
My issue is similar but it is very specific in that the *Companion app does not maintain Mobile app settings when switching networks*

It seems that most of the times when I switch networks (from home WiFi to 4G or WiFi at other places, and back), some account settings are lost. In particular, the following Mobile app settings require constant re-configuration as they seem to reset to defaults:

- **Theme**: set to Dark, always returns to unselected (and displays as Light)

- **Dashboard**: set to my own preferred Dashboard, resets to Overview (default)

Note that other Mobile app settings as well as all the User settings (Time format, etc.) typically remain in place and do not cause an issue. Resetting app (from Debug) and reconfiguring, nor Reset cache, do help, the issue returns again later. Running Companion app version 2025.7.2 (2025.1356) on iPhone Xs Max with iOS  18.6.2.

**Comment by gNeeUs** [comment]
> [@gNeeUs](https://github.com/gNeeUs) i had the same problem, does it keep happening even after changing the network settings i mentioned above?

I will be happy to try, but where exactly are you changing which settings please? Can you provide some more details or possibly a screenshot? Thanks!

**Comment by jefvel** [comment]
@gNeeUs  It's in the system settings:

![image](https://github.com/user-attachments/assets/683c5a23-12f5-4274-8fe1-b96e1f6f6bf2)

![image](https://github.com/user-attachments/assets/8c60c738-ff68-42d6-9bcf-a83c96ddb4d8)

![image](https://github.com/user-attachments/assets/0689bc9f-7362-4037-a233-38147d8d6ba7)

**Comment by gNeeUs** [comment]
> [@gNeeUs](https://github.com/gNeeUs) It's in the system settings:

Thanks! My bad, I was looking in the Companion app which did not have those settings. However, a few remarks:

- I changed those settings with a browser on a computer while logged in to HA as admin. This does allow me to follow your instructions and change the Network settings.
- On my mobile device (iPhone) I use the Companion app, not a browser, and on the Companion app I do not log in as admin, but instead with my personal user name which does not have admin rights. As such, the Settings are not available. Something tells me that the change I made under the admin account may not have any effect on my user account on my mobile, but I will find out over the next few days as I move about.

Cheers!

**Comment by dalester7** [comment]
TL/DR: Check browser_mod settings

I've been having the same issue as the OP for a week now. I recently disabled my primary dashboard (ui Lovelace minimalist) and started using a new dash (I called minimal). I didn't want to have to be tied to minimalist in case the development ever stopped. So everything worked pretty well, but my dash kept reverting to the overview (default). I read through many, many posts about how to possibly fix it... Switched from storage to yaml, renaming to ui-lovelace.yaml, bunch more yaml edits... I was finally (I hope) able to get the setting to stick. I forgot to check browser_mod. There is a setting in there which specifies the default dash to show per user and per device. Mine was pointing at the old minimalist dash that I had disabled. I changed it to my new minimal dash, then verified it was set in settings --> dashboards as the default, lastly verified it was set in user (pic) settings --> default dash. Restarted, verified again. Settings stuck. Before it would reset every time I force closed it on Android. Maybe this will help someone else.

**Comment by sroebert** [comment]
I also thought it was browser mod and even uninstalled it. It fixed that it actually changed the setting of the default dashboard. It did not fix that from time to time it still opens the overview dashboard on iOS, even though it is not even part of the sidebar items. 

I eventually gave up and moved everything to overview.

**Comment by freddeh** [comment]
> TL/DR: Check browser_mod settings

Unfortunately I don’t have that installed and still experience the issue of the default dashboard resetting :(

> I would add that, thinking about it, it might happen most often for me when I'm opening the HA app while away from home and connecting via wireguard, after using it on the local network before - so a network change.

Yes, for me it is change between WiFi and cellular and rebooting HomeAssistant while the app is running in the background.

**Comment by gNeeUs** [comment]
> > [@gNeeUs](https://github.com/gNeeUs) It's in the system settings:
> 
> Thanks! My bad, I was looking in the Companion app which did not have those settings. However, a few remarks:
> 
> * I changed those settings with a browser on a computer while logged in to HA as admin. This does allow me to follow your instructions and change the Network settings.
> * On my mobile device (iPhone) I use the Companion app, not a browser, and on the Companion app I do not log in as admin, but instead with my personal user name which does not have admin rights. As such, the Settings are not available. Something tells me that the change I made under the admin account may not have any effect on my user account on my mobile, but I will find out over the next few days as I move about.
> 

In the meantime, I confirmed the following:

- Changing the network settings as suggested, when done as admin, also affects (all) users. In other words, this can be changed by admin on a computer, and a user with no admin privileges will inherit those settings.
- Have done so, namely disabling the Automatic local network setting, does not solve the issue for me. In the Companion app my default Dashboard as well as the Theme keep being lost frequently.

I can confirm, as several other users have, that this happens on my iPhone anytime a network change takes place. Even at home when I walk around and lose access to WiFi, automatically connect to 4G, then reconnect to the same WiFi and same access point and even reconnect with the same IP on the LAN, the settings go lost.

**Comment by miked1313** [comment]
I think this is a bug with the iOS Companion App ams not a feature request.

I have the same issue.  Regular user (not an admin).  I loose custom default dashboard, custom theme and dark mode preference.  Happens when using the external URL on a mobile network or a different WiFi network.  I have a separate internal URL.  Using my own Cloudflare proxy (not tunnel) in front of HA.

**Comment by LarsStegman** [comment]
@piitaya any chance this could be looked at? If is really annoying. It is also labeled wrong. It's a bug, not a feature request.

**Comment by freddeh** [comment]
> I noticed this happens when switching from wifi to mobile data as mentioned above.
> 
> It seems to have stopped happening if I go into system settings > network, and then uncheck the automatic local address, and click save (repeat twice). The first time it switched back to automatic, but second time it was left unchecked.
> 
> It also seems to work if I then re-enable the automatic address and save.

For me this actually helped a lot. While I had the issue almost twice a week in the past, it did not occur for weeks now. Fingers crossed.

**Comment by austinfavero** [comment]
I'll throw my hat in... this is THE most annoying issue of Home Assistant. It's also THE thing that makes it impossible to get the rest of the family use to using the app. I have to reset my default dashboard multiple times a week. Instead of the round and round circle of trying to find what's clearing the front-end setting... why not alter it to be saved in a more permanent way (server side)??? Please take this issue seriously and address It!

**Comment by burrima** [comment]
Hi, I was able to work-around this limitation by editing the Overview dashboard (click on edit -> take control). By copy-pasting my preferred dashboard's yaml code I could make the Overview board look and behave the same way. Would still be great to select the default dashboard system-wide (on server side).

**Comment by piitaya** [comment]
The issue will be fixed in `2025.12` 🎅 : #27899 

The default panel will now be stored at system level.
Users will be able to set another panel using the profile settings (similar to sidebar)

We will only use the browser storage for non-critical stuff in the future (data-table filters for example)

**Comment by LarsStegman** [comment]
Thank you!

**Comment by austinfavero** [comment]
THANK YOU

**Comment by bbostock** [comment]
Fab, thank you🙏

**Comment by gNeeUs** [comment]
Hallelujah!

And many thanks!

**Comment by GizmoGoody** [comment]
![image](https://github.com/user-attachments/assets/76de9b4b-1fbc-42fd-ac57-fc6f7df3f93b)

**Comment by aglasson** [comment]
Hallelujah!

**Comment by Mucka911** [comment]
Can't wait for this one, so I don't have to use overview as default

**Comment by issleiber** [comment]
The issue still persist with 2026.2.1 ... and newest companion app / or browser. Overwrites the default dashboard.

**Comment by piitaya** [comment]
The default dashboard can now be stored at system level (by going into into config > dashboards). Also, you can override it per user using a settings in your profile. 
What settings are you using?

**Comment by jmbrown79** [comment]
I never had the issue until the Feb update, now the ios app will always open to the new overview tab no matter what settings or changes i try to make to default dashboards for users.  doesn't seem to be an issue with desktop tho

**Comment by frojk** [comment]
> I never had the issue until the Feb update, now the ios app will always open to the new overview tab no matter what settings or changes i try to make to default dashboards for users. doesn't seem to be an issue with desktop tho

I am experiencing the same issue on iOS Companion app on two different iPhones
Both on iOS 26. Was not an issue before 2026.2

**Comment by mhackdo18** [comment]
Still 2026.2.1, even i set my default dashboard in settings and in profile settings, even i tried to install browsermod for default dashboard still it keeps going to overview, and now i dont have controls to overview I cannot paste my rawconfig to overview.

**Comment by nickedw** [comment]
> Still 2026.2.1, even i set my default dashboard in settings and in profile settings, even i tried to install browsermod for default dashboard still it keeps going to overview, and now i dont have controls to overview I cannot paste my rawconfig to overview.

Exactly the same for me on 2 iPads and iphone

**Comment by elliotbeaumont52-bit** [comment]
Exact same issue happens to me

**Comment by mhackdo18** [comment]
If you have Dwaine Dashboard, remove the integration, it has been fix to me

**Comment by issleiber** [comment]
Thanks for the tip … and yes, that’s exactly how it is: Since I've removed the Dwain dashboard, that weird behavior has gone away. Now i can refresh my browser (or companion app) without beeing trapped in "overview" dashboard.

**Comment by nickedw** [comment]
> If you have Dwaine Dashboard, remove the integration, it has been fix to me

Yes, exactly that - completely fixed it. 

That’s been annoying me for a while, although it only started happening a few weeks ago. Great catch anyway. Thankyou.
